File: android/src/main/java/com/witsystem/top/flutterwitsystem/serialport/SerialPortListen.java
package com.witsystem.top.flutterwitsystem.serialport;

/**
 * 串口的回调
 */
public interface SerialPortListen {


    /**
     * 失败
     */
    void serialPortFail(String deviceId, String error, int code);

    /**
     * 成功
     */
    void serialPortSuccess(String deviceId, int code);


    /**
     * 接受到的数据
     */
    void acceptedData(String deviceId, byte[] data);


}

File: ios/Classes/serialport/SerialPortListen.swift
//
// Created by yyjjls on 2020/7/10.
//

import Foundation

/**
 * 串口的回调
 */
protocol SerialPortListen {

    /**
   * 失败
   */
    func serialPortFail(deviceId: String, error: String, code: Int);

    /**
     * 成功
     */
    func serialPortSuccess(deviceId: String, code: Int);


    /**
     * 接受到的数据
     */
    func acceptedData(deviceId: String, data: Data);
}
